config.json

```diff
@@ -9,19 +9,34 @@
     "armhf",
     "armv7"
   ],
-"boot": "auto",
+  "boot": "auto",
   "legacy": true,
+  "map": [
+    "share:rw",
+    "config:rw"
+  ],
   "ports": {
+    "1935/tcp": 1935,
     "8554/tcp": 8554,
-    "1935/tcp": 1935
+    "9997/tcp": 9997
   },
+  "ports_description": {
+    "8554/tcp": "RTSP protocol supports the TCP transport protocol",
+    "1935/tcp": "No idea what this is for.",
+    "9997/tcp": "HTTP api through http://127.0.0.1:9997/v1/paths/list."
+  }
+  ,
   "startup": "application",
   "options": {
+	  "RTSP_API": "yes",
```

Comment:
at least, we can make it optional, as this is not required by integration but an additional feature for add-on